bin/oref0-meal.js

```diff
@@ -21,7 +21,7 @@
 
 var generate = require('oref0/lib/meal');
 function usage ( ) {
-        console.log('usage: ', process.argv.slice(0, 2), '<pumphistory.json> <profile.json> <clock.json> [carbhistory.json]');
+        console.error('usage: ', process.argv.slice(0, 2), '<pumphistory.json> <profile.json> <clock.json> <glucose.json> <basalprofile.json> [carbhistory.json]');
```

Comment:
For future compatibility, in situations like these like to do a ` --with-extra-foo` pattern so that additional options/features can be included without breaking previous optional arguments.